lib/dependencies/ImportParserPlugin.js

```diff
@@ -281,19 +363,25 @@ class ImportParserPlugin {
 
 			const referencedPropertiesInDestructuring =
 				parser.destructuringAssignmentPropertiesFor(expr);
-			if (referencedPropertiesInDestructuring) {
+			const state = getState(parser);
+			const referencedPropertiesInMember = state.get(expr);
+			if (referencedPropertiesInDestructuring || referencedPropertiesInMember) {
 				if (exports) {
-					parser.state.module.addWarning(
-						new UnsupportedFeatureWarning(
-							"`webpackExports` could not be used with destructuring assignment.",
-							/** @type {DependencyLocation} */ (expr.loc)
-						)
+					const warning = new WebpackError(
+						"You don't need `webpackExports` if the usage of dynamic import is statically analyse-able. You can safely remove the `webpackExports` magic comment."
```

Comment:
Do you think we really need it, yeah it can be useful in some cases, but maybe it will be too spammy

/cc @hai-x @xiaoxiaojx

Reply:
I'm ok with that, I can modify once I back to my computer